frontend/src/lib/saveToBookmarks.js

```diff
@@ -0,0 +1,13 @@
+import getBookmarks from './getBookmarks'
+
+export default function saveToBookmarks(trackId) {
+  const currentBookmarks = getBookmarks()
+  const isIdInBookmarks = Boolean(
+    currentBookmarks.find(({ id }) => id === trackId)
+  )
+  const newBookmarks = isIdInBookmarks
+    ? currentBookmarks
+    : [...currentBookmarks, { id: trackId, date: new Date() }]
+
+  localStorage.setItem('bookmarks', JSON.stringify(newBookmarks))
+}
```

Comment:
You know I love backends...
Don't know if you have a user entity in backend, this could be stored there with a manyToMany-Relation so it is not lost with localStorage.

Reply:
Oh, I should delete this unused function. The bookmarks are only stored locally... no backend.